plugins/modules/synchronize.py

```diff
@@ -141,12 +141,17 @@
       - Use the ssh_args specified in ansible.cfg. Setting this to `yes` will also make `synchronize` use `ansible_ssh_common_args`.
     type: bool
     default: no
+  ssh_args:
```

Comment:
This one is provided by the action plugin as well (it depends on `use_ssh_args`). It probably should not be documented.

Reply:
Yes. Sure.